docs/source/dataframe-sql.rst

```diff
@@ -30,12 +30,20 @@ The following packages may be of interest
 - `blazingSQL`_, part of the Rapids project, implements SQL queries using ``cuDF``
   and Dask, for execution on CUDA/GPU-enabled hardware, including referencing
   externally-stored data
+  
+- `dask-sql`_, a new project (started August 2020), adds a SQL query layer on top
+  of Dask. The API matches blazingSQL but it uses CPU instead of GPU.
```

Comment:
As a personal philosophy I always try to avoid anything that is prone to obsolescence
```suggestion
- `dask-sql`_ adds a SQL query layer on top of Dask.
  The API matches blazingSQL but it uses CPU instead of GPU.
```